rocket-basicauth/src/lib.rs

```diff
@@ -47,7 +47,7 @@ impl PartialEq<String> for Username {
     }
 }
 
-#[derive(Debug, PartialEq)]
+#[derive(Debug, PartialEq, Clone)]
```

Comment:
We can revert this once we use https://docs.rs/rocket/0.5.0-rc.1/rocket/struct.Rocket.html#method.state.